sunpy/data/data_manager/tests/test_manager.py

```diff
@@ -88,9 +88,10 @@ def override_file_tester(manager):
         # Inside the file is replaced
         data_function(override_file_tester)
 
-    with manager.override_file('test_file', f'{folder}/another_file'):
-        # Inside the file is replaced
-        data_function(override_file_tester)
+    # TODO: this combined with the check above fails on windows
```

Comment:
@vn-ki you got any ideas why this file would duplicate here with a `.0` extension on windows?